python/lib/physiological.py

```diff
@@ -215,8 +215,46 @@ def get_parameter_type_id(self, parameter_name):
                 get_last_id  = True
             )
 
+            # link the parameter_type_id to a parameter type category
+            category_id = self.get_parameter_type_category_id()
+            self.db.insert(
+                table_name   = 'parameter_type_category_rel',
+                column_names = ('ParameterTypeCategoryID', 'ParameterTypeID'),
+                values       = (category_id, parameter_type_id),
+                get_last_id  = False
+            )
+
         return parameter_type_id
 
+    def get_parameter_type_category_id(self):
+        """
+        Greps ParameterTypeCategoryID from parameter_type_category table.
+        If no ParameterTypeCategoryID were found, the script will create it in
+        parameter_type_category.
+
+        :return: ParameterTypeCategoryID
+         :rtype: int
+        """
+
+        category_result = self.db.pselect(
+            query='SELECT ParameterTypeCategoryID '
+                  'FROM parameter_type_category '
+                  'WHERE Name = %s ',
+            args=('Electrophysiology Variables',)
+        )
+        if category_result:
+            category_id = category_result[0]['ParameterTypeCategoryID']
+        else:
+            # if no results, create an entry in parameter_type_category
```

Comment:
I believe that it would be better to insert that value in a one time script executed during the update of LORIS than here. Even if you have to create a related PR on LORIS.

This way, no check would be necessary except maybe to raise and exception/error.

Reply:
@PapillonMcGill I removed the code creating that category here. I already sent the SQL patch to create the category on the LORIS side so all should be good.

Thanks!